docs/spec_draft.md

```diff
@@ -405,6 +448,80 @@ operation.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.sine
+
+`stablehlo.sine(operand) -> result`
+
+### Semantics
+
+Performs element-wise sine operation on `operand` tensor and produces a `result` tensor.
+
+### Operands
+
+| Name | Type |
+|-|-|
+| `operand` | tensor of floating-point or complex element types |
+
+### Results
+
+| Name | Type |
+|-|-|
+| `result` | tensor of floating-point or complex element types |
+
+### Constraints
+
+  * Supported shapes: all static shapes.
+  * `result` must have the same type as that of `operand`.
+
+### Examples
+
+```mlir
+// %operand: [
+              [0.0, 1.57079632],       // [0, pi/2]
+              [3.14159265, 4.71238898] // [pi, 3pi/2]
+             ]
+%result = stablehlo.sine %operand : tensor<2x2xf32>
+// %result: [[0.0, 1.0], [0.0, -1.0]]
+```
+
+[Back to Ops](#index-of-documented-ops)
+
+## stablehlo.tanh
+
+`stablehlo.tanh(operand) -> result`
+
+### Semantics
+
+Performs element-wise tanh operation on `operand` tensor and produces a `result` tensor.
+$$tanh(x)=\frac{e^{x}-e^{-x}}{e^{x}+e^{-x}}$$
```

Comment:
What should be our policy for providing explanations on what some of these operations do? How about we consistently refer to IEEE-754 if they are part of it, or express them in terms of IEEE-754 operations if they aren't?

E.g. `tanh` is called out in IEEE-754, so we could just say "Performs element-wise tanh operation on `operand` tensor and produces a `result` tensor, according to the IEEE-754 specification"?